src/nvim/memline.c

```diff
@@ -3472,40 +3474,26 @@ static char *findswapname(buf_T *buf, char **dirp, char *old_fname, bool *found_
             // If vimrc has "simalt ~x" we don't want it to
             // interfere with the prompt here.
             flush_buffers(FLUSH_TYPEAHEAD);
-          }
 
-          if (swap_exists_action != SEA_NONE && choice == SEA_CHOICE_NONE) {
-            const char *const sw_msg_1 = _("Swap file \"");
-            const char *const sw_msg_2 = _("\" already exists!");
-
-            const size_t fname_len = strlen(fname);
-            const size_t sw_msg_1_len = strlen(sw_msg_1);
-            const size_t sw_msg_2_len = strlen(sw_msg_2);
-
-            const size_t name_len = sw_msg_1_len + fname_len + sw_msg_2_len + 5;
-
-            char *const name = xmalloc(name_len);
-            memcpy(name, sw_msg_1, sw_msg_1_len + 1);
-            home_replace(NULL, fname, name + sw_msg_1_len, fname_len, true);
-            xstrlcat(name, sw_msg_2, name_len);
-            int dialog_result
-              = do_dialog(VIM_WARNING,
-                          _("VIM - ATTENTION"),
-                          name,
-                          proc_running
-                          ? _("&Open Read-Only\n&Edit anyway\n&Recover\n&Quit\n&Abort")
-                          : _("&Open Read-Only\n&Edit anyway\n&Recover\n&Delete it\n&Quit\n&Abort"),
-                          1, NULL, false);
-
-            if (proc_running && dialog_result >= 4) {
+            if (swap_exists_action != SEA_NONE) {
```

Comment:
Why is the check for `choice == SEA_CHOICE_NONE` from `SwapExists` autocommand removed?